src/mbgl/gl/bytes.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+namespace mbgl {
+namespace gl {
+
+struct ElementOffset {
+    std::size_t value;
```

Comment:
Why? If you want to match `GLsizei`, it should be `int32_t`, or actually `uint16_t` because this must be a non-negative value.